README.md

```diff
@@ -110,6 +110,8 @@ That should be all you need to do.
 
 ## Deploying via Docker
 
+> In order to invalidate the client-side cache, be sure to bump the `API_VERSION` in `webpack.config.js` prior to each deploy.
```

Comment:
One small suggestion: maybe reference the env var rather than bumping the hard-coded `API_VERSION` in the webpack config?  That would allow us to decouple content updates from requiring changes to the `scaife-viewer` code base.

Reply:
Updated. I added a new info endpoint to return the API version. On the client, I am now automatically invalidating the cache every 24 hours. It can manually be invalidated as well by bumping the API version.